mu4/userscores/qml/MuseScore/UserScores/internal/TempoSettings.qml

```diff
@@ -0,0 +1,160 @@
+import QtQuick 2.9
+import QtQuick.Controls 2.2
+import QtQuick.Layouts 1.3
+
+import MuseScore.Ui 1.0
+import MuseScore.UiComponents 1.0
+import MuseScore.UserScores 1.0
+
+FlatButton {
+    id: root
+
+    height: 96
+
+    property var model: null
+
+    property var arrowX
+    property var popupPositionX
+    property var popupPositionY: height
+    property alias oppened: popup.visible
+
+    backgroundColor: oppened ? ui.theme.accentColor : ui.theme.buttonColor
+
+    TempoView {
+        anchors.horizontalCenter: root.horizontalCenter
+        anchors.verticalCenter: root.verticalCenter
+
+        tempoNote: MusicalSymbolIconCode.CROTCHET // TODO: get from model
```

Comment:
I think it's OK to keep it here

Reply:
This value depends on the selected note in the popup and the value must be saved in the model.